pub_sub/go/sdk/order-processor/app.go

```diff
@@ -18,7 +19,13 @@ var sub = &common.Subscription{
 }
 
 func main() {
-	s := daprd.NewService(":6001")
+	//read app-port passed thru command line
+	appPort, isSet := os.LookupEnv("APP_PORT")
+	if !isSet {
+		log.Fatalf("--app-port is not set")
```

Comment:
The error message should mention that the environment variable was not set. It is set by passing the `--app-port` option to Dapr, but this message can be misleading for users. We can additionally add a hint to the error message that this environment variable is set by Dapr. Thoughts?